scripts/reformat.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+find . -type f -name '*.go' -print0 | xargs -0 gofmt -s -e -l -w #-s to try to simpifly code, -e to print errors, -w to write improved version to actual file
```

Comment:
should the `find` cmd  point to another location where the *.go files root directories are?

Reply:
concerning that the script is called from the make file in the root directory, . should cover the entire project ?

Reply:
correct, leave it like this. One more thing, you could improve the `find` by making use of an absolute path instead of the `.`, as in this [script](https://github.com/homeport/havener/blob/develop/scripts/build-binaries.sh#L25) , it is just more elegant.